resource-manager/src/main/java/org/snomed/module/storage/RF2Row.java

```diff
@@ -0,0 +1,40 @@
+package org.snomed.module.storage;
+
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class RF2Row {
+    private final Map<Integer, String> columns = new HashMap<>();
+    private boolean found;
+    private String metadataResourcePath;
```

Comment:
@pgwilliams Class represents an RF2 row, but I needed `found` and `metadataResourcePath` for conditional processing. This sort of breaches the idea/scope of an RF2 row but I couldn't think of another way of doing it.